### PR TITLE
docs(shielded-scan) Improve feature docs

### LINKED ISSUE #8046: Document how to run and query the results of the blockchain scanning MVP
## Motivation

We want a "naive" zebra user to be able to use `zebra-scan` to query the Zcash blockchain with a viewing key and get a list of the transactions related with it

## Scope

The documentation should provide details of:
- how to compile and run zebra with the `zebra-scan` feature enabled
- how to specify the viewing key(s), e.g location and format for the config file containing the keys
- how to query the database to retrieve data about the transactions found

It would be nice if we could also post the same instructions to the blog and include screenshots and examples, e.g using the ZECPages viewing key.

The incomplete draft document is here:
https://github.com/ZcashFoundation/zebra/blob/main/book/src/user/shielded-scan.md

### LINKED ISSUE #8049: Check if any of the security issues are blockers for the MVP, and document the remaining security issues

### PR DESCRIPTION
**Pull request by oxarbitrage**
## Motivation

Improve documentation for the shielded-scan feature.

Close https://github.com/ZcashFoundation/zebra/issues/8049 and close https://github.com/ZcashFoundation/zebra/issues/8046

#### Check before marking the PR as ready for review:
  - [x] Will the PR name make sense to users?
  - [x] Does the PR have a priority label?
  - [ ] Have you added or updated tests?
  - [x] Is the documentation up to date?

## Solution

Improve documentation.

## Review

Anyone can review.

### Reviewer Checklist

Check before approving the PR:
  - [ ] Does the PR scope match the ticket?
  - [ ] Are there enough tests to make sure it works? Do the tests cover the PR motivation?
  - [ ] Are all the PR blockers dealt with?
        PR blockers can be dealt with in new tickets or PRs.

_And check the PR Author checklist is complete._

### CODE REVIEW

File: book/src/user/shielded-scan.md [hunk 1]
```diff
@@ -9,28 +9,82 @@ Ongoing development is tracked in issue [#7728](https://github.com/ZcashFoundati
...
 - the `[shielded_scan.sapling_keys_to_scan]` table for diversifiable full viewing keys.
 
-TODO: add a definition for DFVK, link to its format, and add examples and links to keys and database settings.
+Sapling diversifiable/extended full viewing keys strings start with `zxviews` as described in [ZIP-32](https://zips.z.cash/zip-0032#sapling-extended-full-viewing-keys).
+
+For example, to scan for transactions with the [public ZECpages viewing key](https://zecpages.com/z/all) use:
+
+```toml
+[shielded_scan.sapling_keys_to_scan]
+"zxviews1q0duytgcqqqqpqre26wkl45gvwwwd706xw608hucmvfalr759ejwf7qshjf5r9aa7323zulvz6plhttp5mltqcgs9t039cx2d09mgq05ts63n8u35hyv6h9nc9ctqqtue2u7cer2mqegunuulq2luhq3ywjcz35yyljewa4mgkgjzyfwh6fr6jd0dzd44ghk0nxdv2hnv4j5nxfwv24rwdmgllhe0p8568sgqt9ckt02v2kxf5ahtql6s0ltjpkckw8gtymxtxuu9gcr0swvz" = 1
```
Comment (teor2345) [inline book/src/user/shielded-scan.md]: Let's explain that the number here is the birthday height?

File: book/src/user/shielded-scan.md [hunk 2]
```diff
@@ -9,28 +9,82 @@ Ongoing development is tracked in issue [#7728](https://github.com/ZcashFoundati
...
+```
 
 ## Running Sapling Scanning
 
-Launch Zebra and wait for 12-24 hours.
```
Comment (teor2345) [inline book/src/user/shielded-scan.md:29]: It's helpful to tell users they will need to wait a day or so for full scanning results.

File: book/src/user/shielded-scan.md [hunk 3]
```diff
@@ -9,28 +9,82 @@ Ongoing development is tracked in issue [#7728](https://github.com/ZcashFoundati
...
 
-## Expert: Querying Raw Sapling Scanning Results
+Querying results this way requires a certain level of expertise.
 
-TODO: Copy these instructions and examples here:
-- https://github.com/ZcashFoundation/zebra/issues/8046#issuecomment-1844772654
+## Quering results with the `scanning-results-reader` utility.
 
-Database paths are different on Linux, macOS, and Windows.
+Another easier way to read the results of found transactions and more is to use the utility tool named `scanning-results-reader`. Please read the [tool description](https://github.com/ZcashFoundation/zebra/tree/main/zebra-utils#scanning-results-reader) to learn how to use it. Instructions are straightforward.
```
Comment (teor2345) [inline book/src/user/shielded-scan.md]: Many people try instructions in the order that they are written, so I'd suggest putting this paragraph at the start of the querying section.

**Review by teor2345 — DISMISSED** [review]
Looks good, just a few tweaks to answer common questions or make it easier to read.

File: README.md [hunk 4]
```diff
@@ -141,6 +141,7 @@ You can also build Zebra with additional [Cargo features](https://doc.rust-lang.
 - `prometheus` for [Prometheus metrics](https://zebra.zfnd.org/user/metrics.html)
 - `sentry` for [Sentry monitoring](https://zebra.zfnd.org/user/tracing.html#sentry-production-monitoring)
 - `elasticsearch` for [experimental Elasticsearch support](https://zebra.zfnd.org/user/elasticsearch.html)
+- `shielded-scan` for [experimental shielded scan support](https://zebra.zfnd.org/user/shielded-scan.md)
```
Comment (upbqdn) [inline README.md]:
```suggestion
- `shielded-scan` for [experimental shielded scan support](https://zebra.zfnd.org/user/shielded-scan.html)
```

File: book/src/user/shielded-scan.md [hunk 5]
```diff
@@ -9,28 +9,86 @@ Ongoing development is tracked in issue [#7728](https://github.com/ZcashFoundati
...
 ## Configuration
 
 In `zebrad.toml`, use:
 - the `[shielded_scan]` table for database settings, and
 - the `[shielded_scan.sapling_keys_to_scan]` table for diversifiable full viewing keys.
 
-TODO: add a definition for DFVK, link to its format, and add examples and links to keys and database settings.
+Sapling diversifiable/extended full viewing keys strings start with `zxviews` as described in [ZIP-32](https://zips.z.cash/zip-0032#sapling-extended-full-viewing-keys).
+
+For example, to scan for transactions with the [public ZECpages viewing key](https://zecpages.com/z/all) use:
```
Comment (upbqdn) [inline book/src/user/shielded-scan.md]:
```suggestion
For example, to scan for transactions with the [public ZECpages viewing key](https://zecpages.com/boardinfo) use:
```

File: book/src/user/shielded-scan.md [hunk 6]
```diff
@@ -9,28 +9,86 @@ Ongoing development is tracked in issue [#7728](https://github.com/ZcashFoundati
...
+...
+```
+
+The Zebra scanner will resume the task if your Zebra instance went down for any reason. In a new start, Zebra will display:
+
+```
+Last scanned height for key number 0 is 1798000, resuming at 1798001
+```
+
+## Quering results with the `scanning-results-reader` utility.
```
Comment (upbqdn) [inline book/src/user/shielded-scan.md]:
```suggestion
## Querying results with the `scanning-results-reader` utility.
```

**Review by upbqdn — APPROVED** [review]

File: book/src/user/shielded-scan.md (hunk 1 above)
Comment (upbqdn) [reply book/src/user/shielded-scan.md]: I'll do that in a subsequent PR.

File: book/src/user/shielded-scan.md [hunk 7]
```diff
@@ -9,28 +9,86 @@ Ongoing development is tracked in issue [#7728](https://github.com/ZcashFoundati
...
 - the `[shielded_scan.sapling_keys_to_scan]` table for diversifiable full viewing keys.
 
-TODO: add a definition for DFVK, link to its format, and add examples and links to keys and database settings.
+Sapling diversifiable/extended full viewing keys strings start with `zxviews` as described in [ZIP-32](https://zips.z.cash/zip-0032#sapling-extended-full-viewing-keys).
+
+For example, to scan for transactions with the [public ZECpages viewing key](https://zecpages.com/z/all) use:
+
+```toml
+[shielded_scan.sapling_keys_to_scan]
+"zxviews1q0duytgcqqqqpqre26wkl45gvwwwd706xw608hucmvfalr759ejwf7qshjf5r9aa7323zulvz6plhttp5mltqcgs9t039cx2d09mgq05ts63n8u35hyv6h9nc9ctqqtue2u7cer2mqegunuulq2luhq3ywjcz35yyljewa4mgkgjzyfwh6fr6jd0dzd44ghk0nxdv2hnv4j5nxfwv24rwdmgllhe0p8568sgqt9ckt02v2kxf5ahtql6s0ltjpkckw8gtymxtxuu9gcr0swvz" = 1
```
Comment (upbqdn) [inline book/src/user/shielded-scan.md]:
```suggestion
"zxviews1q0duytgcqqqqpqre26wkl45gvwwwd706xw608hucmvfalr759ejwf7qshjf5r9aa7323zulvz6plhttp5mltqcgs9t039cx2d09mgq05ts63n8u35hyv6h9nc9ctqqtue2u7cer2mqegunuulq2luhq3ywjcz35yyljewa4mgkgjzyfwh6fr6jd0dzd44ghk0nxdv2hnv4j5nxfwv24rwdmgllhe0p8568sgqt9ckt02v2kxf5ahtql6s0ltjpkckw8gtymxtxuu9gcr0swvz" = 419200
```

**Review by upbqdn — DISMISSED** [review]

**Review by upbqdn — APPROVED** [review]

**Pull request merged**